.github/workflows/ci.yml

```diff
@@ -37,11 +37,18 @@ jobs:
             args: --all --all-targets
             components: clippy
           - name: test-fvm
-            command: test
-            args: --package fvm --no-default-features
+            covname: fvm-lcov.info
+            command: llvm-cov
+            args: --package fvm --no-default-features --lcov --output-path fvm-lcov.info
+            components: llvm-tools-preview
           - name: test
+            covname: lcov.info
+            command: llvm-cov
+            args: --all --exclude fvm --exclude fvm_conformance_tests --exclude fvm_integration_tests --exclude "*actor" --lcov --output-path lcov.info
+            components: llvm-tools-preview
+          - name: integration
             command: test
-            args: --all --exclude fvm --exclude fvm_conformance_tests
+            args: --package fvm_integration_tests --package "*actor"
```

Comment:
- Any reason we're excluding the actor packages?
- Any reason not to count the integration tests towards test coverage?

Reply:
- actor tests try to build to wasm via `wasm-builder` and breaks the coverage tool so getting coverage reporting them are more complex (will likely need to edit some cargo.toml) than I wanted to have for this PR
- integration tests build actors

These should still be added, but I wanted to push this PR through first so we can start getting metrics. If you feel like it should all happen in one PR then I can do that instead.

Reply:
Makes sense.